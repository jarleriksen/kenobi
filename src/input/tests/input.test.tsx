import React from 'react';
import { render } from '@testing-library/react';
import { Input } from '../';
import '@testing-library/jest-dom/extend-expect';

describe('Input', () => {
  test('renders correctly', () => {
    const { getByTestId, asFragment } = render(<Input name="test-input" />);

    const input = getByTestId('kenobi-input');

    expect(input).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
