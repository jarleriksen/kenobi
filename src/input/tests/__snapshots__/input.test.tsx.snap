// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
<DocumentFragment>
  <input
    class="sc-bdVaJa kOtoB"
    data-testid="kenobi-input"
    name="test-input"
  />
</DocumentFragment>
`;
